### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#659] [Q&A]
Can someone please explain what n and s are in underlined parts?
![image](https://user-images.githubusercontent.com/89933766/224931240-6b2e3ec7-1a52-4077-86fe-93da67ac6049.png)

**Comment by m-spitfire** [comment]
They're the names of variables that you can use in that branch of match expression. `n` gets the value of `name` and `s` similarly.

**Comment by KihongHeo** [comment]
Good. Let me close this issue.
